react/Widget.tsx

```diff
@@ -8,15 +8,17 @@ interface Props {
   theme: string
   width: string
   height: string
+  stars: string
 }
 
 const Widget: StorefrontFunctionComponent<Props> = ({
   // 5419b6a8b0d04a076446a9ad = micro review count
   templateId = '5419b6a8b0d04a076446a9ad',
-  businessUnitId,
-  theme,
-  width,
-  height,
+  businessUnitId = '',
+  theme = 'light',
+  width = '100%',
+  height = '240px',
+  stars = '4,5',
```

Comment:
This would change the default values for everyone using the app, is this desirable? Wouldn't it be a breaking change?

Reply:
it's true. i remove that's not necessary.